README.md

```diff
@@ -52,7 +52,7 @@ Make changes to code at two places for the new URL that Heroku has produced, as
        ........
        "repository": {
        "type": "git",
-       "url": "https://newURL-from-heroku.herokuapp.com/"
+       "url": "https://<newURL-from-heroku>.herokuapp.com/“
```

Comment:
It looks like this is the wrong quotation mark character at the end of the line. Perhaps a setting in your editor? Or just a typo? I don't see the problem on any of the other lines you changed.

Reply:
Yes! Just a typo. Fixed.